### DISCUSSION TITLE
Are stop events returned by `getMarkers()`?

### DISCUSSION THREAD

**Discussion by justDeeevin** [mjansen4857/pathplanner#308] [Q&A] [answered]
I have a path with events, but all of the events are at stop points rather than markers. Inspecting the `.path` file (it's just json), I find that the `markers` array is empty. If I make a new `FollowPathWithEvents` command and pass in `path.getMarkers()` for the `pathMarkers` constructor argument, will the command execute the events listed at the stop points?

**Comment by mjansen4857** [comment, marked as answer]
No. Stop events can only be used with an auto builder.
